README.md

```diff
@@ -57,48 +60,253 @@ This is work in progress as of 14 Nov 2018:
 [![License](https://img.shields.io/npm/l/@airtasker/spot.svg)](https://github.com/airtasker/spot/blob/master/package.json)
 
 <!-- toc -->
-* [Usage](#usage)
-* [Commands](#commands)
-<!-- tocstop -->
+
+- [Usage](#usage)
+- [Commands](#commands)
+  <!-- tocstop -->
+
 # Usage
 
 To get started and set up an API declaration in the current directory, run:
+
 ```
 npx @airtasker/spot init
 ```
 
 You can then run a generator with:
+
 ```
 npx @airtasker/spot generate --api api.ts
 ```
 
+## `@api`
+
+Define an API. This is required and must only be defined once:
+
+```TypeScript
+import { api } from "@airtasker/spot";
+
+@api({
```

Comment:
Now that `@api()` is used separately from `@endpoint()`, the syntax looks a bit unnecessarily complicated.

Something to address separately: maybe we actually just want to expose a conventional function `declareApi()`, so you do the following:
```typescript
declareApi({
  name: "My API",
  description: "My really cool API"
});
```

Reply:
Adding test-interactions will force us to potentially introduce some new syntax - which may or may not work well with the current syntax. It makes sense at that point that we evaluate the syntax overall to ensure both consistency and intuitiveness.